projects/backend/__tests__/application.spec.ts

```diff
@@ -11,12 +11,23 @@ const stub = (req: Request, res: Response) => {
     res.sendStatus(200)
 }
 
+const stubEditionController = {
+    READ: (req: Request, res: Response) => {
+        console.log(`endpoint (READ): ${req.path} called`)
```

Comment:
I think we have request logging for this all so might not need this fine detail logging